python/ray/dashboard/dashboard.py

```diff
@@ -333,15 +333,15 @@ async def kill_actor(self, req) -> aiohttp.web.Response:
             self.dashboard_controller.kill_actor(actor_id, ip_address, port))
 
     async def logs(self, req) -> aiohttp.web.Response:
-        hostname = req.query.get("hostname")
+        ip = req.query.get("ip_address")
         pid = req.query.get("pid")
-        result = self.dashboard_controller.get_logs(hostname, pid)
+        result = self.dashboard_controller.get_logs(ip, pid)
         return await json_response(self.is_dev, result=result)
 
     async def errors(self, req) -> aiohttp.web.Response:
-        hostname = req.query.get("hostname")
+        ip = req.query.get("ip_address")
         pid = req.query.get("pid")
-        result = self.dashboard_controller.get_errors(hostname, pid)
+        result = self.dashboard_controller.get_errors(ip, pid)
```

Comment:
These calls have corresponding API requests in the front-end that will also need to be changed that exist in `api.ts`.

Reply:
I see, that makes sense.  Your concerns about continuing make sense; I'm happy with whatever you and/or @edoakes decide.